modules/platforms/dotnet/Apache.Ignite.Core.Tests/Services/ServicesTest.cs

```diff
@@ -870,20 +870,6 @@ public void TestCallJavaService()
                 binSvc.testBinaryObject(
                     Grid1.GetBinary().ToBinary<IBinaryObject>(new PlatformComputeBinarizable {Field = 6}))
                     .GetField<int>("Field"));
-            
```

Comment:
Why!?

Reply:
Thanks, I've reverted changes. This was an incorrect merge with the master branch.